ports/stm32/mpconfigboard_common.h

```diff
@@ -217,7 +217,7 @@
 #define PYB_EXTI_NUM_VECTORS (24)
 #define MICROPY_HW_MAX_I2C (4)
 #define MICROPY_HW_MAX_TIMER (17)
-#define MICROPY_HW_MAX_UART (8)
+#define MICROPY_HW_MAX_UART (9)
```

Comment:
might be clearer to do something like this:
```c
#define MICROPY_HW_MAX_LPUART (1)
#define MICROPY_HW_MAX_UART (8)
```
(ie last line there unchanged), and then in `mpconfigport.h`:
```c
struct _pyb_uart_obj_t *pyb_uart_obj_all[MICROPY_HW_MAX_UART + MICROPY_HW_MAX_LPUART]; \
```